config/exchanges/FI_SE-SE1.yaml

```diff
@@ -1,6 +1,6 @@
 capacity:
-  - -1500
-  - 1100
+  - -1900
+  - 1900
```

Comment:
Where did you get these numbers from?

According to https://www.fingrid.fi/en/grid/construction/aurora-line/

It adds:
> Aurora Line will increase the cross-border capacity between the countries approximately by 900 MW in the direction from Finland to Sweden and 800 MW in the direction from Sweden to Finland.